src/Config/ObjectModel/McpRuntimeOptions.cs

```diff
@@ -4,33 +4,60 @@
 using System.Text.Json.Serialization;
 using Azure.DataApiBuilder.Config.Converters;
 
-namespace Azure.DataApiBuilder.Config.ObjectModel;
-
-/// <summary>
-/// Global MCP endpoint runtime configuration.
-/// </summary>
-public record McpRuntimeOptions
+namespace Azure.DataApiBuilder.Config.ObjectModel
 {
-    public const string DEFAULT_PATH = "/mcp";
+    /// <summary>
+    /// Runtime configuration options for MCP (Model Context Protocol)
+    /// </summary>
+    public record McpRuntimeOptions
+    {
+        public const string DEFAULT_PATH = "/mcp";
 
-    [JsonPropertyName("enabled")]
-    public bool Enabled { get; init; }
+        /// <summary>
+        /// Whether MCP endpoints are enabled
+        /// </summary>
+        [JsonPropertyName("enabled")]
+        public bool Enabled { get; init; } = true;
 
-    [JsonPropertyName("path")]
-    public string? Path { get; init; }
+        /// <summary>
+        /// The path where MCP endpoints will be exposed
+        /// </summary>
+        [JsonPropertyName("path")]
+        public string? Path { get; init; } = DEFAULT_PATH;
```

Comment:
why is the `Path` property nullable if it has a DEFAULT value? Will it ever be null? I understand the constructor argument `path` could be null if the config file doesnt have this property.